libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -577,8 +577,10 @@ bool AP_Arming::rc_arm_checks(AP_Arming::Method method)
             }
         }
 
-        if (rc().arming_check_throttle()) {
-            const RC_Channel *c = rc().channel(rcmap->throttle() - 1);
+        // only check throttle inputs if not skipped and not flying
+        if (rc().arming_check_throttle() && !AP_Notify::flags.flying) {
```

Comment:
I still want to always do this, it's the same as the way the RPY checks all work. We need these to be consistent.

Reply:
I guess we forgot to discuss the in-flight check. I guess that's between you and @tridge at this point.
I agree that they should be consistent.

Reply:
we have a bypass for watchdog armed anyway, so lets drop the flying check here